<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sACD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="BrowserPage" name="browserPage"/>
      <widget class="WorkPage" name="workPage"/>
      <widget class="PhotoWidget" name="photoPage"/>
      <widget class="MetadataEditorPage" name="metadataEditorPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCopyFile"/>
    <addaction name="actionMoveFile"/>
    <addaction name="actionDeleteFile"/>
    <addaction name="actionRenameFile"/>
    <addaction name="actionMkDir"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowMetadata"/>
    <addaction name="actionShowPreview"/>
    <addaction name="actionShowSecondBrowser"/>
    <addaction name="actionShowHidden"/>
    <addaction name="actionShowIcons"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionZoom_1_1"/>
     <addaction name="actionZoomToFit"/>
    </widget>
    <widget class="QMenu" name="menuOrientation">
     <property name="title">
      <string>Orientation</string>
     </property>
     <addaction name="actionRotateCW"/>
     <addaction name="actionRotateCCW"/>
     <addaction name="actionResetTransformations"/>
    </widget>
    <addaction name="actionFullScreen"/>
    <addaction name="actionWork_Page"/>
    <addaction name="menuZoom"/>
    <addaction name="menuOrientation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMetadataEditor"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionShowMetadata">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Metadata</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionShowPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Preview</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionShowSecondBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 2nd Browser</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionCopyFile">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy File</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMoveFile">
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move File</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionRenameFile">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDeleteFile">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete File(s)</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionShowHidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Hidden</string>
   </property>
   <property name="toolTip">
    <string>Show Hidden Files</string>
   </property>
  </action>
  <action name="actionShowIcons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Icons</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="text">
    <string>Toggle Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionWork_Page">
   <property name="text">
    <string>Toggle Work Page</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionZoom_1_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom 1:1</string>
   </property>
  </action>
  <action name="actionZoomToFit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom to fit</string>
   </property>
  </action>
  <action name="actionRotateCW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Clockwise</string>
   </property>
  </action>
  <action name="actionRotateCCW">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Counterclockwise</string>
   </property>
  </action>
  <action name="actionResetTransformations">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Original State</string>
   </property>
  </action>
  <action name="actionMetadataEditor">
   <property name="text">
    <string>Metadata Editor</string>
   </property>
   <property name="toolTip">
    <string>Metadata Editor</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionMkDir">
   <property name="text">
    <string>Create Dir</string>
   </property>
   <property name="toolTip">
    <string>Create New Directory</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BrowserPage</class>
   <extends>QWidget</extends>
   <header>browser_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorkPage</class>
   <extends>QWidget</extends>
   <header>work_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhotoWidget</class>
   <extends>QWidget</extends>
   <header>photo_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MetadataEditorPage</class>
   <extends>QWidget</extends>
   <header location="global">metadata_editor_page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
